os/fs/semaphore/sem_close.c

```diff
@@ -159,6 +158,10 @@ int sem_close(FAR sem_t *sem)
 
 	inode_semgive();
 	return OK;
+
+	#endif							/* CONFIG_FS_NAMED_SEMAPHORES */
```

Comment:
1. need to remove indentation here
2. When the config is enabled, this function has two returns without any conditional. This will give build warning or  SVACE defect or something else.